### PR TITLE
Move Push Notification constant out of WordPressComApi class.

### PR DESCRIPTION
**Pull request by SergioEstevao**
Refs #5245 

To test:
 - Check if push notifications are getting to the app when events happen in the backend: new comment, new post, etc...

Needs review: @jleandroperez

### CODE REVIEW

File: WordPress/Classes/Networking/NotificationsServiceRemote.swift [hunk 1]
```diff
@@ -8,6 +8,16 @@ import UIDeviceIdentifier
...
 {
+    #if DEBUG
+        static let WordPressComApiPushAppId = "org.wordpress.appstore.dev"
+    #else
+    #if INTERNAL_BUILD
+        static let  WordPressComApiPushAppId = "org.wordpress.internal"
+    #else
+        static let WordPressComApiPushAppId = "org.wordpress.appstore"
+    #endif
+    #endif
```
Comment (jleandroperez) [inline WordPress/Classes/Networking/NotificationsServiceRemote.swift]: Can we please please move this to a "central" location with more visibility? (perhaps `Constants.h/m`).

Comment (SergioEstevao) [inline WordPress/Classes/Networking/NotificationsServiceRemote.swift]: Does any other part of the code needs to know about this settings? Or is this something that will need to change in the future?

Comment (jleandroperez) [inline WordPress/Classes/Networking/NotificationsServiceRemote.swift]: AFAIK nope, but mostly for visibility + maintainability.

Comment (SergioEstevao) [inline WordPress/Classes/Networking/NotificationsServiceRemote.swift]: @jleandroperez ready for another look.

**Comment by jleandroperez** [comment]
Thanks for the changes @SergioEstevao!!

:shipit:

**Pull request merged**